include/tallies/regular_cartesian_mesh_filter.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef REGULAR_CARTESIAN_MESH_FILTER_H
+#define REGULAR_CARTESIAN_MESH_FILTER_H
+
+#include <tallies/cartesian_filter.hpp>
+#include <utils/position.hpp>
+
+#include <array>
+
+class RegularCartesianMeshFilter : public CartesianFilter {
+ public:
+  RegularCartesianMeshFilter(Position r_low_, Position r_high_, size_t nx_,
+                             size_t ny_, size_t nz_);
+
+  ~RegularCartesianMeshFilter() = default;
+
+  StaticVector3 get_indices(const Tracker& tktr) override final;
+
+  std::vector<TracklengthDistance> get_indices_tracklength(
+      const Tracker& trkr, double d_flight) override final;
+
+  // Note that the method given is applicable when the reduced_dimension is
+  // used.
```

Comment:
I put this intentionally, because in case, if any dimension has only 1 bin, so it will not return the value.

But, I think, it is causing a confusion so I will remove it.